v2/thirdparty/cors.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: cors
+title: CORS
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./thirdparty/cors.mdx -->
+
+import GuidesLink from "/src/components/guidesLink"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
+import NodeJSFrameworkSubTabsServerless from "/src/components/tabs/NodeJSFrameworkSubTabsServerless"
+import PythonFrameworkSubTabs from "/src/components/tabs/PythonFrameworkSubTabs"
+import GoFrameworkSubTabs from "/src/components/tabs/GoFrameworkSubTabs"
+import AppInfoForm from "/src/components/appInfoForm"
+import CoreInjector from "/src/components/coreInjector"
+import {Question, Answer}from "/src/components/question"
+import BackendSDKCasing from "/src/components/BackendSDKCasing"
+
+# CORS
+
```

Comment:
The sub headings i suggest:
- What is CORS?
- Why and when is it needed?
- Why does SuperTokens require you to have CORS setup?
- How to enable CORS in your app?
- Troubleshooting common issues